modules/kg-schemas/src/test/resources/data/nexus/core/instance/v0.1.0/all-fields.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    {
+      "@vocab": "https://bbp-nexus.epfl.ch/vocabs/nexus/core/v0.0.1/",
+      "value": "schema:value",
+      "unitCode": "schema:unitCode"
+    }
+  ],
+  "@type": "Instance",
+  "@id": "{{base}}/data/orgId/domainId/domainId/v0.0.1/d76df1e4-d57f-4614-b18c-a378087e524f",
```

Comment:
The id is inconsistent: missing `schemaName`. Also please use `v0.1.0` as the initial version.